src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import print_function
+import re
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.expect import pexpect_ssh
+from constants import ds_instance_name
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.defaultdebuglevel
+class TestDefaultDebugLevel(object):
+    """ Check sssd default debug level """
+    @pytest.mark.tier1
+    def test_0001_check_default_debug_level(self, multihost, backupsssdconf):
+        """
+        @Title: IDM-SSSD-TC: Check default debug level by authenticating the
+        user
+        """
+        section = "domain/%s" % ds_instance_name
+        domain_params = {'debug_level': ''}
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf(section, domain_params, action='delete')
+        tools.clear_sssd_cache()
+        user = 'foo1@%s' % ds_instance_name
+        client = pexpect_ssh(multihost.client[0].sys_hostname, user,
+                             'Secret123', debug=False)
+        client.login(login_timeout=30, sync_multiplier=5,
+                     auto_prompt_reset=False)
+        log_list = ['sssd.log', 'sssd_example1.log', 'sssd_nss.log',
+                    'sssd_pam.log', 'sssd_implicit_files.log']
+        for log in log_list:
+            log = '/var/log/sssd/%s' % log
+            log_str = multihost.client[0].get_file_contents(log).decode(
+                'utf-8')
+            find1 = re.compile(r'0x0020')
+            find2 = re.compile(r'0x0010')
+            find3 = re.compile(r'0x0040')
+            find4 = re.compile(r'Starting with debug level = 0x0070')
+            if not (find1.search(log_str) or find2.search(log_str) or
+                    find3.search(log_str)) and find4.search(log_str):
+                status = 'FAIL'
+            else:
+                status = 'PASS'
+        assert status == 'PASS'
```

Comment:
This was you always verify 'status' of last entry in 'log_list'. Other results are just lost (over-written in the for-loop)